src/domobserver.js

```diff
@@ -82,6 +82,17 @@ export class DOMObserver {
     this.currentSelection.set(this.view.root.getSelection())
   }
 
+  ignoreSelectionMutation(sel) {
+    let desc = this.view.docView.nearestDesc(sel.anchorNode, true)
+
+    if (desc.node.isText) return false
+
+    return desc && desc.dom && desc.ignoreMutation({
```

Comment:
What is the `desc.dom` check for?

Reply:
The `target` property below needs to be a DOM node, so I added a check to make sure it's set - if it's guaranteed to always be there then this check can be removed.

Reply:
I guess what we really want as target (and as argument to `nearestDesc`) is the first element node that contains both the `anchorNode` and `focusNode`. (So that if one side of the selection stays in a widget, but the other escapes it, we don't ignore the selection change.)